data-model/SDP-sample-records/person_sample1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/person/1",
+  "label": "John bar Me'sam (6th c. CE)",
+  "uniform_name": "John bar Me'sam, Monk, 6th c. CE",
+  "authority_files": [],
+  "dates": {
+    "birth": {},
+    "death": {},
+    "floruit": {"date_human-readable": "6th c. CE", "date_normalized": "0500/0600"}
+  },
+  "sex": "male",
+  "associated_places": ["https://sinaimanuscripts.librar.ucla.edu/place/1"],
+  "associated_persons": [],
+  "references": [],
+  "contributor": ["gkessel"]
+}
```

Comment:
Fields we could add:

- we could add a description or abstract
- we could designate a primary role? (author, scribe, illustrator, patron, etc.) (the full list of roles could easily be computed from other data)
- there are a lot of computed fields we could have: attested_roles, authored_works, manuscript_attestations, etc. but I'm not sure if those need to be contained in the record or if they would be computed via a user query or when we go to output a data dump like for SMDL or JSON-LD?